statusnet/statusnet.php

```diff
@@ -742,8 +743,10 @@ function statusnet_cron(App $a, array &$b)
 	}
 	logger('statusnet: cron_start');
 
+	/// @TODO Use PConfig::get() here maybe?
```

Comment:
No, this is fetching all the users who set StatusNet mirroring. it starts from a pconfig value to get user IDs, which is the opposite of PConfig::get()